### DISCUSSION TITLE
关于搜索

### DISCUSSION THREAD

**Discussion by Doddddd** [LibChecker/LibChecker#492] [Q&A] [answered]
从搜索结果页面返回会直接退出app，而不是退出搜索，是bug么？

**Comment by qhy040404** [comment, marked as answer]
我没复现出来，我从搜索结果出来还是主界面，有录屏吗

**Reply by Doddddd** [reply]
https://user-images.githubusercontent.com/88972478/204978038-7ebc5665-108d-47e2-873e-52306007d158.mp4

**Reply by qhy040404** [reply]
哪个版本

**Reply by Doddddd** [reply]
https://user-images.githubusercontent.com/88972478/204978308-9c47a089-b695-460c-824f-3270f9056c67.mp4

**Reply by Doddddd** [reply]
2.3.4.7c927c8a (1448)

**Reply by qhy040404** [reply]
感觉是oneui的奇怪bug

**Reply by qhy040404** [reply]
[app-release.zip](https://github.com/LibChecker/LibChecker/files/10131754/app-release.zip)
你在这个app里面直接触发返回看看keycode

**Reply by Doddddd** [reply]
![Screenshot_20221201_192921_checkkey.png](https://user-images.githubusercontent.com/88972478/205041660-8754584d-d45e-4213-91a0-0457fa1308c8.png)

**Reply by qhy040404** [reply]
keycode没问题啊

**Reply by qhy040404** [reply]
你是安卓13吗

**Reply by Doddddd** [reply]
是的

**Reply by Doddddd** [reply]
已经修复了，谢谢
